tenant/config/manager/manager.yaml

```diff
@@ -44,7 +50,7 @@ spec:
       containers:
       - command:
         - /manager
-        image: controller:latest
+        image: vrgf2003/tenant:latest
```

Comment:
We should be able to use the k8s gcr I'd think. @tashimi are you aware of the process to use it/configure full CI/CD of our repo? At the least we should create an issue to configure this asap.

Reply:
I am going to change it to Multitenancy/tenant for now and I suggest @tashimi to create a docker hub account to save the image instead of using gcr.  Countries outside of US may have difficulty to access gcr.